docs/end-user-topics/ORCID.rst

```diff
@@ -18,7 +20,7 @@ The process is quite simple. If you are logged in now, you can `click here`_ to
 From here, you'll be redirected to ORCID. With just a few clicks, you will now see this on the link services page of Dockstore.
 
 .. image::  /assets/images/docs/orcid_after_account_linkage.png
-  :alt: After linking ORCID to Dockstore there will be options to relink, unlick, or view your token
+  :alt: After linking ORCID to Dockstore there will be options to relink, unlink, or view your token
```

Comment:
Was more amusing before but I'll accept it :grin:

Reply:
If there's one thing I learned from watching my roommate's dog is that you cannot unlick that which has been licked.

![Photo on 2-1-22 at 8 50 AM](https://user-images.githubusercontent.com/27784612/152013627-254492a7-9fa8-4102-8a6a-abf691ffc47c.jpg)